installation/windows.md

```diff
@@ -85,4 +85,103 @@ There is currently no automatic update script for Windows. To update manually, d
 
 ## Starting openHAB as a service
 
-(work in progress...)
+To install the openHAB process as a service in Windows in order to be able to launch it automatically and run it in the background, follow these steps:
+
+1. Complete the [prerequisites](#prerequisites) and regular [installation](#installation) steps, including the package selection
+2. Issue the following 2 commands in your openHAB console:
+```
+feature:install service-wrapper
+wrapper:install --name "openHAB2" --display "openHAB2" --description "openHAB 2 Service"
+```
+![Wrapper Install_Windows](http://imageshack.com/a/img922/1700/teZEiW.jpg)
```

Comment:
I wish I knew how to do it...
I open up a trial account every time on these crappy providers to host the images 👎